velox/connectors/hive/storage_adapters/s3fs/tests/MinioServer.h

```diff
@@ -47,14 +47,22 @@ class MinioServer {
     return tempPath_->path;
   }
 
-  std::shared_ptr<const Config> hiveConfig() const {
-    const std::unordered_map<std::string, std::string> config({
+  typedef std::unordered_map<std::string, std::string> map_type;
```

Comment:
map_type name is very generic; it would be nice to avoid this typedef if possible

Reply:
I removed this now as the default constructor got simplified above.